packages/expo-sqlite/ios/CRSQLiteLoader.m

```diff
@@ -1,17 +1,17 @@
 // Copyright 2015-present 650 Industries. All rights reserved.
 
 #import <ExpoSQLite/CRSQLiteLoader.h>
-#import <sqlite3/sqlite3.h>
+#import <ExpoSQLite/sqlite3.h>
 
 int crsqlite_init_from_swift(sqlite3 *db) {
-  sqlite3_enable_load_extension(db, 1);
+  exsqlite3_enable_load_extension(db, 1);
```

Comment:
This reminds me. We should remove crsqlite. It's no longer maintained. Matt has moved on to other things

Reply:
yes, we've talked this with Brent. we would remove it from sdk 53 and give a deprecated warning in sdk 52
https://linear.app/expo/issue/ENG-12984/deprecate-enablecrsqlite-and-warn-when-used